extensions/cleanshotx/src/annotate.tsx

```diff
@@ -0,0 +1,9 @@
+import open from "open";
+import {closeMainWindow } from "@raycast/api";
```

Comment:
have you run `rpm run lint`? I'm pretty sure prettier should rewrite this to `import { closeMainWindow } from ...`

Reply:
> have you run `rpm run lint`? I'm pretty sure prettier should rewrite this to `import { closeMainWindow } from ...`

Weird, linter didn't catch that, (perhaps my linter is not acting sanely)  here's my before and after!
![AShot-2022 01 03-16 14 16@2x](https://user-images.githubusercontent.com/43297314/147981048-b83c79e0-4835-48ef-9a90-d424ff482fea.png)

Anyways, I've added and removed spaces as you requested!